apps/miiverse-api/src/services/api/routes/communities.ts

```diff
@@ -315,7 +284,7 @@ router.post('/', multer().none(), async function (request: express.Request, resp
 
 	const ownedSubcommunityCount = await Community.countDocuments(ownedQuery);
 	if (ownedSubcommunityCount >= 4) {
-		return respondCommunityError(response, 401, 911);
+		return badRequest(response, ApiErrorCode.CREATE_TOO_MANY_COMMUNITIES, 403);
```

Comment:
I see a lot of changes in error codes. The most common ones:
- 401 unauthorized changed to 403 forbidden
- 422 changed to more semantically correct status codes

Are these changes all intentional? Are we sure this won't cause issues with consoles, do they care about the status codes?

Reply:
They are intentional (e.g. 403 is because we're already past the authentication middleware by now, so Unauthorized is the wrong code) but I'm not sure how the clients react. We can at least expect different pop-up error codes: https://github.com/PretendoNetwork/miiverse-api/wiki/Miiverse-Error-Codes
I smoketested it and things still worked, but this is the sort of thing where edge cases could come up (maybe a client is expecting a 404 on a certain community)

Reply:
I've done additional testing with Nintendo Land (apparently one of the pickier titles) and I wasn't able to get it to do anything weird on the error paths; though it's still pretty limited and challenging to hit all the paths

Reply:
Good that you gave it some extra tests. For the rest we'll just test in production, we can do a rollback if it's *too* unstable.